Lib/fontTools/ttLib/tables/_c_m_a_p.py

```diff
@@ -164,7 +164,7 @@ def decompile(self, data, ttFont):
 		if ttFont.lazy is False:  # Be lazy for None and True
 			self.ensureDecompiled()
 
-	def ensureDecompiled(self):
+	def ensureDecompiled(self, recurse=None):
```

Comment:
like for glyf, I think you want `recurse=False`, not `None` (see my other comment related to glyf.ensureDecompiled)

Reply:
I used None because the argument is unused.